test/dotnet-new3.UnitTests/CommonTemplatesTests.cs

```diff
@@ -115,9 +115,10 @@ public void AllCommonProjectsCreateRestoreAndBuild(string expectedTemplateName,
 Processing post-creation actions\.\.\.
 Running 'dotnet restore' on ({finalProjectName})\.\.\.
   Determining projects to restore\.\.\.
-  Restored ({finalProjectName}) \(in \d{{1,3}} ms|\d(\.\d{{1,3}}){{0,1}} sec\)\.
+(.*$\n)*  Restored ({finalProjectName}) \(in \d{{1,5}}(\.\d{{1,5}}){{0,1}} \w*\)\.
```

Comment:
not sure if it is related to this PR

Reply:
This PR also increases the SDK version, which breaks the tests.